### PR TITLE
Issue #361 voting open

### PR DESCRIPTION
**Pull request by baob**
WIP

### CODE REVIEW

File: app/views/home/index.html.haml [hunk 1]
```diff
@@ -1,5 +1,10 @@
-- if swapping_open?
-  = render partial: "swaps_open"
+- if voting_open?
+  = render partial: "voting_open"
 
 - else
-  = render partial: "swaps_closed"
+  - $stderr.puts "voting_open? ------ #{voting_open?}"
```
Comment (aspiers) [inline app/views/home/index.html.haml]: Remove the debug ;-)

Comment (baob) [reply app/views/home/index.html.haml]: oops yes thank you

Comment (baob) [reply app/views/home/index.html.haml]: Done

**Comment by baob** [comment]
Present state of affairs if VOTING_OPEN is true ...
image

**Pull request merged**